### DISCUSSION TITLE
bug: Unable to import 'read_zbarray

### DISCUSSION THREAD

**Discussion by d-strom** [modflowpy/flopy#1870] [Q&A] [answered]
Hi,

For unknown reason I cannot import read_zbarray from flopy.utils. I'm trying to import an already run zonebudget to be able to plot and tabulate the results. It's a MF6 divs-model. From what i gather in the documentation one way to do it is throught the operation in the image below. I dont really know why the package seems to be unavailable, any ideas?

![image](https://github.com/modflowpy/flopy/assets/74509984/6f583e88-e0c7-46e8-87f5-e7a7d77a21fe)

**Comment by wpbonelli** [comment, marked as answer]
Hi @DStrom1987, the `ZoneBudgetOutput` class was deprecated in version 3.3.5. There is now a `ZoneBudget` class with a `get_dataframes()` method.
